python_Docker_build/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.9
+
+RUN git clone https://github.com/tncowart/python
```

Comment:
I think you should have `WORKDIR /` before this line, to be explicit about the current directory.